pkg/operator/starter.go

```diff
@@ -39,15 +53,22 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
-	kubeInformers := v1helpers.NewKubeInformersForNamespaces(kubeClient, util.TargetNamespace, util.OperatorNamespace, util.UserSpecifiedGlobalConfigNamespace)
+	kubeInformers := v1helpers.NewKubeInformersForNamespaces(kubeClient,
+		"",
+		util.TargetNamespace,
+		util.OperatorNamespace,
+		util.UserSpecifiedGlobalConfigNamespace,
+		util.InfraNamespace,
+		metav1.NamespaceSystem,
```

Comment:
New additions to this list:

- Cluster scoped (`""`)
- `openshift-infra`
- `kube-system`